signer/src/main.rs

```diff
@@ -37,7 +39,12 @@ struct SignerArgs {
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     // Initialize logging
-    sbtc::logging::setup_logging("info,signer=debug", false);
+    // TODO: The whole logging thing should be revisited. We should support
+    //   enabling different layers, i.e. for pretty console, for opentelem, etc.
```

Comment:
Sure, can you create and link an issue for this?

Reply:
Linked #497